server/db/models/Order.js

```diff
@@ -0,0 +1,35 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Order = db.define('order', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  date: {
+    type: Sequelize.STRING,
```

Comment:
I think there's a Sequelize DATE type?

Reply:
Sequelize date was kind of annoying to work with, using the isDate validate on STRING makes sure that it adheres to the dd/mm/yyyy format regardless.

Reply:
gotcha